src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,145 @@
+import static seedu.addressbook.common.Messages.MESSAGE_WELCOME;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Scanner;
+
+import seedu.addressbook.commands.CommandResult;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+/**
+ * Text UI of the application.
+ */
+public class Formatter {
+
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+
+    private static final String DIVIDER = "===================================================";
+
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+
+    /** Format of a comment input line. Comment lines are silently consumed when reading user input. */
+    private static final String COMMENT_LINE_FORMAT_REGEX = "#.*";
+
+    /**
+     * Returns true if the user input line should be ignored.
+     * Input should be ignored if it is parsed as a comment, is only whitespace, or is empty.
+     *
+     * @param rawInputLine full raw user input line.
+     * @return true if the entire user input line should be ignored.
+     */
+    public boolean shouldIgnore(String rawInputLine) {
+        return rawInputLine.trim().isEmpty() || isCommentLine(rawInputLine);
+    }
+
+    /**
+     * Returns true if the user input line is a comment line.
+     *
+     * @param rawInputLine full raw user input line.
+     * @return true if input line is a comment.
+     */
+    public boolean isCommentLine(String rawInputLine) {
+        return rawInputLine.trim().matches(COMMENT_LINE_FORMAT_REGEX);
+    }
+
+    public String getCommandFormat() {
+        return LINE_PREFIX + "Enter command: ";
+    }
+
+    public String getCommandEnteredFormat(String fullInputLine) {
+        return "[Command entered:" + fullInputLine + "]";
+    }
+
+    public void showWelcomeMessage(String version, String storageFilePath) {
+        TextUi textUi = new TextUi();
+        String storageFileInfo = String.format(MESSAGE_USING_STORAGE_FILE, storageFilePath);
+        textUi.showToUser(DIVIDER,
+                DIVIDER,
+                MESSAGE_WELCOME,
+                version,
+                MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE,
+                storageFileInfo,
+                DIVIDER);
+
+    }
+
+    public void showGoodbyeMessage() {
+        TextUi textUi = new TextUi();
+        textUi.showToUser(MESSAGE_GOODBYE, DIVIDER, DIVIDER);
+    }
+
+
+    public void showInitFailedMessage() {
+        TextUi textUi = new TextUi();
+        textUi.showToUser(MESSAGE_INIT_FAILED, DIVIDER, DIVIDER);
+    }
```

Comment:
I think there shouldn't be any `showToUser` method in Formatter class because that's the point of SRP.